components/status_dropdown/index.jsx

```diff
@@ -25,15 +27,14 @@ function mapStateToProps(state) {
     }
 
     const userId = currentUser.id;
-    const ctz = getUserTimezone(state, userId);
-    const etz = areTimezonesEnabledAndSupported(state);
+
     return {
         userId,
```

Comment:
userId can be simplified as well.

```
userId: currentUser.id
```

Reply:
@michaelgamble something strange is happening but when i do userId: currentUser.id, i am getting error
```
scheduler.development.js:178 Uncaught ReferenceError: userId is not defined
    at Function.mapStateToProps [as mapToProps] (index.jsx:31)
    at mapToPropsProxy (wrapMapToProps.js:41)
    at Function.detectFactoryAndVerify (wrapMapToProps.js:50)
    at mapToPropsProxy (wrapMapToProps.js:41)
    at handleFirstCall (selectorFactory.js:22)
    at pureFinalPropsSelector (selectorFactory.js:63)
    at connectAdvanced.js:319
    at mountMemo (react-dom.development.js:15442)
    at Object.useMemo (react-dom.development.js:15738)
    at useMemo (react.development.js:1521)
```